python/cppyy/__init__.py

```diff
@@ -396,7 +396,7 @@ def typeid(tt):
         return _typeids[tt]
     except KeyError:
         tidname = 'typeid_'+str(len(_typeids))
-        gbl.cling.runtime.gCling.ProcessLine(
+        cppexec(
```

Comment:
@wlav, I mostly wanted to check if that change is ok.

Reply:
I'm a bit surprised that `gCling` is still passed in these calls:

```
gbl.InterOp.Evaluate(gbl.cling.runtime.gCling
```

is there ever a case where `Evaluate` does not operate on gCling, or a case where there are multiple `gCling` objects in the same process? Or plans for such?

Reply:
Here the `gCling` is still a placeholder for the pointer to the interpreter that we have created. `InterOp` takes an interpreter pointer to avoid storing a global interpreter instance. In theory it should work with different interpreter if needed but I don't think there are any plans for that. It is just to avoid extra global state.